src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/SupplierController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
+{
+    [Route("api/v1/orders/{orderId}/supplier")]

```

Comment:
This should also be `/sections/supplier`.

Reply:
👍